IndicoToolkit/Extractions.cs

```diff
@@ -119,7 +182,7 @@ public void removeHumanAddedPredictions()
             for (int i = 0; i < Preds.Count; i++) {
```

Comment:
I think we should remove `removeHumanAddedPredictions` from both c# and python toolkit. If a user highlights a label and then manually adjusts the predicted value, out current logic would incorrectly flag it as not manually added.

Reply:
the idea is to remove predictions that don't have indexes so that you don't erroneously try to apply positional logic to them-- but I'm down to listen to your argument on the call later today, it's possible this isn't needed